.github/workflows/ci.yml

```diff
@@ -131,14 +141,30 @@ jobs:
       - name: Extract Artifacts
         run: ./.github/extract-artifacts.sh
 
+      - name: Cache Turbo
+        uses: actions/cache@v2
+        with:
+          path: "**/.turbo"
+          key: "turbo-${{ github.ref }}-${{ hashFiles('packages/**/src/**') }}-${{ github.sha }}"
+          restore-keys: |
+            turbo-${{ github.ref }}-${{ hashFiles('packages/**/src/**') }}-${{ github.sha }}
+            turbo-${{ github.ref }}-${{ hashFiles('packages/**/src/**') }}-
+            turbo-${{ github.ref }}-
+
       - name: Install NPM Dependencies
-        run: yarn install --prefer-offline --frozen-lockfile --ignore-engines
+        run: yarn install --prefer-offline --frozen-lockfile --ignore-engines --registry https://registry.npmjs.org --network-timeout 300000
         env:
           CI: true
 
       - name: Test
+        if: ${{ matrix.os != 'windows-latest' }}
         run: yarn test
 
+      # Turbo seems to make the tests hang on Windows, just run them directly
+      - name: Test (Windows)
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: yarn workspace astro run test
```

Comment:
Couldn't seem to get `yarn test` (which runs `yarn turbo run test`) to exit on Windows. I suspect it might have something to do with our mocha test timeout?

For now, skip using `turbo` on Windows.